text/0030-standardized-sender-sig.md

```diff
@@ -0,0 +1,137 @@
+- Feature Name: Standardized Sender Signature
+- Start Date: 2022-03-14
+- MCIP PR: [mobilecoinfoundation/mcips#0030](https://github.com/mobilecoinfoundation/mcips/pull/0030)
+- MobileCoin Epic: None
+
+# Summary
+[summary]: #summary
+
+Enact MCIP #18 (Janus Mitigation), which sets the ephemeral key in the encrypted fog hint
+to be the tx private key times the Ristretto base point. The original motivation was to enact
+Janus attack mitigation.
+
+We find a secondary use-case to motivate this: it becomes possible for the signer to create
+a Schnorr signature over any payload, which a verifier can validate against the keys appearing
+in a `TxOut`, without revealing the identity of the signer. We propose to both do #18 and standardize
+the construction and validation of this signature.
+
+# Motivation
+[motivation]: #motivation
+
+Often in a payments flow, it is necessary for the sender and recipient to exchange additional messages
+and metadata about the payment. Frequently, these messages need to be authenticated.
+
+We have a mechanism for doing this in the memo field, which attaches fixed-size payloads to TxOut's.
+However, this imposes fairly strict size requirements on the data that can be sent.
+
+Outside of the memo field, we have a notion of "receipts" and "confirmation numbers" which a sender
+can use to identify themselves to a recipient. These are hashes of the TxOut shared secret, which
+are similarly deniable, and convince the recipient that whoever sent them this number must have access
+to secrets involved in the construction of the TxOut.
+
+However, these numbers are useless for convincing a third-party who is not a party to the transaction,
```

Comment:
```suggestion
However, these numbers are only capable of convincing a third-party who also has access to the recipients' private keys,
```

Or something like this - indicating that a 3rd party could be enlisted to do these confirmations, but only if they are provided with the correct material to do the validation.

Reply:
fair!

Reply:
i feel like, it is a bit weird to phrase things this way, because the recipient is not ever expected to give away their private keys, and if they are, that's an exceptional situation

Reply:
Good point - and that's one of our motivating design constraints. The "numbers are useless for convincing a third-party" felt false in that if I really needed to convince the third party, I could provide them my private keys. Maybe rephrasing to say we don't want to put users in a situation where they need to provide their private keys to a third party is a design goal of MobileCoin?

Reply:
thank you!